FlowCrypt/Functionality/Error Handling/KeyServiceErrorHandler.swift

```diff
@@ -10,14 +10,37 @@ import UIKit
 
 enum CreateKeyError: Error {
     case weakPassPhrase(_ strength: CoreRes.ZxcvbnStrengthBar)
-    // Missing user email
+    /// Missing user email
     case missedUserEmail
```

Comment:
All instances of `missed ...` should be all `missing ...`

`missedUserEmail` looks like broken English to me

I'll file another issue for that.